domain/model/monster_model.go

```diff
@@ -0,0 +1,9 @@
+package model
+
+type Monster struct {
+	Id              string
```

Comment:
Even if it seems redundant, it's very useful to have the json tags for each of your fields

Reply:
I would recommend changing a bit the data types of your fields here, I noticed that the ID is actually an integer